eng/intellisense.targets

```diff
@@ -1,40 +1,46 @@
-<Project InitialTargets="VerifyAssemblySupportsDocsXmlGeneration">
+<Project>
 
   <PropertyGroup>
-    <UseIntellisensePackageDocXmlFile Condition="'$(UseIntellisensePackageDocXmlFile)' == ''">true</UseIntellisensePackageDocXmlFile>
+    <UseCompilerGeneratedDocXmlFile Condition="'$(UseCompilerGeneratedDocXmlFile)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Extensions.'))">true</UseCompilerGeneratedDocXmlFile>
   </PropertyGroup>
 
-  <Target Name="VerifyAssemblySupportsDocsXmlGeneration"
-          Condition="'$(UseIntellisensePackageDocXmlFile)' != 'true'">
```

Comment:
Let me see if I understand this: You're now inverting the the check for `UseCompilerGeneratedDocXmlFile` in line 7 (before line 15) which means we can disable simply skip the compiler-generated doc xml file logic in parital facade assemblies and in PNSE assemblies. Correct?